pkgs/by-name/ll/llm-ls/package.nix

```diff
@@ -19,21 +24,38 @@ rustPlatform.buildRustPackage {
     sha256 = "sha256-ICMM2kqrHFlKt2/jmE4gum1Eb32afTJkT3IRoqcjJJ8=";
   };
 
-  cargoHash = "sha256-Fat67JxTYIkxkdwGNAyTfnuLt8ofUGVJ2609sbn1frU=";
+  cargoHash = "sha256-m/w9aJZCCh1rgnHlkGQD/pUDoWn2/WRVt5X4pFx9nC4=";
+
+  cargoPatches = [
+    # https://github.com/huggingface/llm-ls/pull/102
+    ./fix-time-compilation-failure.patch
+    # https://github.com/huggingface/llm-ls/commit/479401f3a5173f2917a888c8068f84e29b7dceed
```

Comment:
Not sure this comment is really doing much, given the URL (with .patch on the end) is the one used for fetchpatch.